### ISSUE TITLE
Expand/Collapse icon on a selected tree item has insufficient contrast against its background

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#860]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![7 1](https://user-images.githubusercontent.com/22068434/88343845-2d774100-ccf7-11ea-8c35-4c2bc36b2f32.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1383.01, all 6 modes are now compliant
